setup.sh

```diff
@@ -47,6 +47,14 @@ die "Xcode version for the repository should be at least ${repository_xcode_vers
 
 REPO_ROOT=$(git rev-parse --show-toplevel)
 
+if [[ -n "${CI-}" ]]; then
+    echo "Running on CI, skipping git lfs install"
+else
+    echo "ℹ️  Running git lfs install..."
+    git lfs install
```

Comment:
is there a way to check git-lfs is installed?

right now I am getting :
```
› ./setup.sh                                                                                                                                                                                                                         08:50:37
ℹ️  Running git lfs install...
WARNING: You called a Git command named 'lfs', which does not exist.
Continuing in 0.1 seconds, assuming that you meant 'log'.
fatal: ambiguous argument 'install': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```

also we can add it to readme :
`brew install git-lfs`

Reply:
> check for git-lfs

done: 8007fc8a0f1d6e6fcbaa63c1b7bb196388ea21aa

> update readme

done: 6e6496ba047aba50407b4992f8e0a9ef57be4f49